_config_base.yml

```diff
@@ -47,10 +47,10 @@ release_info:
     start_time: 2018-06-17 15:10:52.34274101 +0000 UTC
   v19.2:
     name: v19.2.0
-    version: v19.2.0-alpha.20190606
+    version: v19.2.0-alpha.20190701
```

Comment:
Just ICYMI: Date needs to be updated throughout. Apologies if it was already on your to-do list.

Reply:
Just confirmed, turns out we're sticking with `20190701`